src/Avatar.js

```diff
@@ -1,20 +1,31 @@
-import React from 'react';
+import React, {useState} from 'react';
 
 import './App.css';
 
 export function Avatar({avatar, name, onClickAvatar}) {
-    let initials = name && name.split(`${~name.indexOf(' ') ? ' ' : ''}`, 2).map(n => n[0]).join('').toUpperCase();
+    const [isErrorLoadImg, setIsErrorLoadImg] = useState(false);
+
+    let initials = name && name.trim().split(' ', 2).map(n => n[0]).join('').toUpperCase();
 
     return (
         <div onClick={onClickAvatar} class="avatar flex-center flex-column">
-            {(avatar && <img class="avatar-image" src={avatar} alt=""></img>)
-            || (initials && <div class="avatar-initials flex-center">{initials}</div>)
-            || (
-                <>
-                    <div class="symbol-head" />
-                    <div class="symbol-shoulders" />
-                </>
-            )}
+            {(avatar && !isErrorLoadImg 
+                && (
+                    <img 
+                        class="avatar-image" 
+                        src={avatar} 
+                        alt=""
+                        onError={() => {setIsErrorLoadImg(true);}}
+                    />
+                )) || (
+                    initials && <div class="avatar-initials flex-center">{initials}</div>
+                ) || (
+                    <>
+                        <div class="symbol-head" />
+                        <div class="symbol-shoulders" />
+                    </>
+                )
+            }
```

Comment:
I would move it to another component and use `is else` to render stuff, now it's a mess